Frontend/speedcart-react/src/components/PageLayout/PageLayout.tsx

```diff
@@ -0,0 +1,18 @@
+// src/components/PageLayout.tsx
+import React from 'react';
+import StatusModal from '@components/StatusModal';
+
+import styles from './PageLayout.module.css';
+
+const PageLayout = ({ children, status = null, showStatusModal = false, modalProps = null, className=''}) => {
```

Comment:
Get rid of `showStatusModal` prop; you should only need to check if the `modalProps` is not null here, so there's no need for another prop indicating if a `StatusModal` component should exist here.